docs/en/guides/getting-started.md

```diff
@@ -0,0 +1,181 @@
+3. Access to a filesystem where you can store project files and a pipeline configuration. On the filesystem, create a "projects” folder and a "configs” folder as a proper directory setup. 
+4. A configuration folder named "the_other_side.” Inside the "configs” folder where you store your configs, create a folder using this project name. We will be creating this project in this exercise.
+ 
+  
+## About the Advanced Project Setup Wizard
+
+The Advanced Project Setup Wizard in Shotgun Desktop generates the configuration necessary to support customizations to meet the needs of your production pipeline. Using the Advanced Project Setup Wizard, you can extend the Default Configuration creating a configuration specific to a project’s proprietary pipeline needs. 
+
+The Default Configuration includes: 
+* A basic filesystem schema and templates that determine where files live on disk 
+* All of the supported [software integrations](https://support.shotgunsoftware.com/hc/en-us/articles/219039798-Integrations-Apps-and-Engines) allowing file management from directly inside the user’s software applications. 
+
+Customizations are only limited by imagination, cleverness, and programming knowledge or the ability to borrow from what others in the Shotgun community have created.
+ 
+## Extending the Default Configuration 
+
+A Configuration is required for every project. The first time a project is accessed through Shotgun Desktop a Basic Configuration is generated by Shotgun Toolkit. This Basic Configuration automatically detects the supported software applications a user has on their system. The integrations that support displaying project related information in the applications and the Apps to manage files related to the project are managed through the pipeline configuration. This information is displayed through the integrated display [Panel]( https://support.shotgunsoftware.com/hc/en-us/articles/219033098-Shotgun-Panel). Publishes are managed by publishing files with the [Publisher]( https://support.shotgunsoftware.com/hc/en-us/articles/219032998-Publishing-your-work). Uploading files is managed through the [Loader]( https://support.shotgunsoftware.com/hc/en-us/articles/219033078-Load-Published-Files-). The Basic Configuration does not include filesystem management or the templates for specifying how files and directories are named. It also doesn’t have the plethora of Apps and hooks, or the ability to customize settings that allow for extending a pipeline configuration. The Advanced Configuration Wizard generates a Default Configuration that allows customizations and has more Apps, hooks and reference guides to support you on your journey. This guide will expand your mind to the possibilities. 
+
+## Difference between the Default Configuration and Basic Configuration.
+
+![alt text](../../images/guides/getting-started/1_config_comparison.png "Configuration comparison chart")
```

Comment:
we cannot have text inside images like this - we need to be able to translate the content.